### PR TITLE
Fixed routes and minor bugs

### PR DESCRIPTION
**Pull request by Genemichael25**

### CODE REVIEW

File: app/javascript/components/components/Footer.js [hunk 1]
```diff
@@ -1,18 +1,17 @@
 import React, { Component } from "react"
-import { NavLink } from 'react-router-dom'
 
 class Footer extends Component {
   render() {
     return (
       <>
         <footer>
-          <NavLink className={"about-us"} to={`/aboutus`}>
+          <a className={"about-us"} href={`/aboutus`}>
```
Comment (elysemontano) [inline app/javascript/components/components/Footer.js]: I would recommend using NavLink from react-router-dom here as this is navigating internally on this app.

File: app/javascript/components/pages/Home.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ class Home extends Component {
         </div>
       </div>
     </>
-
+  )
```
Comment (elysemontano) [inline app/javascript/components/pages/Home.js]: Check the indentation here.

**Review by elysemontano — CHANGES_REQUESTED** [review]
It looks like ProtectedTripIndex was removed from App.js.  Is that correct?
Left a few other comments to look at as well.

File: app/javascript/components/components/Footer.js (hunk 1 above)
Comment (Genemichael25) [reply app/javascript/components/components/Footer.js]: Switched back to NavLink, we were experimenting with a tags when we couldn't get not found to show when signed in. We had moved header and footer outside the router.

**Review by elysemontano — APPROVED** [review]
Approving, but double-check if ProtectedTripIndex is supposed to be in your App.js before merging.

**Pull request merged**